js/modules/k6/html/elements_gen.go

```diff
@@ -176,18 +245,18 @@ func (e HrefElement) Download() string {
 func (e HrefElement) ReferrerPolicy() string {
 	attrVal := e.attrAsString("referrerpolicy")
 	switch attrVal {
-	case "no-referrer":
-		return "no-referrer"
-	case "no-referrer-when-downgrade":
-		return "no-referrer-when-downgrade"
-	case "origin":
-		return "origin"
-	case "origin-when-cross-origin":
-		return "origin-when-cross-origin"
-	case "unsafe-url":
-		return "unsafe-url"
-	default:
-		return ""
+	case constno_referrer:
+		return constno_referrer
+	case constno_referrer_when_downgrade:
+		return constno_referrer_when_downgrade
+	case constorigin:
+		return constorigin
+	case constorigin_when_cross_origin:
+		return constorigin_when_cross_origin
+	case constunsafe_url:
+		return constunsafe_url
+	default: 
+		return constBlank 
```

Comment:
Instead of all this, just do:

```go
switch attrVal {
  case "no-referrer", "no-referrer-when-downgrade", ...:
    return attrVal
  default:
    return ""
}
```